src/settings-ui/Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -216,197 +198,269 @@
+                                                    FontFamily="{StaticResource SymbolThemeFontFamily}"
+                                                    Foreground="{ThemeResource InfoBarErrorSeverityIconBackground}"
+                                                    Glyph="{StaticResource InfoBarIconBackgroundGlyph}"
+                                                    Visibility="{x:Bind ShowBadgeOnPluginSettingError}" />
+                                                <FontIcon
+                                                    FontFamily="{StaticResource SymbolThemeFontFamily}"
+                                                    Foreground="{ThemeResource InfoBarErrorSeverityIconForeground}"
+                                                    Glyph="{StaticResource InfoBarErrorIconGlyph}"
+                                                    Visibility="{x:Bind ShowBadgeOnPluginSettingError}" />
+                                            </Grid>
+
+                                            <ToggleSwitch
+                                                x:Uid="PowerLauncher_EnablePluginToggle"
+                                                IsOn="{x:Bind Path=Disabled, Converter={StaticResource BoolNegationConverter}, Mode=TwoWay}" />
+                                        </StackPanel>
+                                        <labs:SettingsExpander.Items>
+                                            <labs:SettingsCard
+                                                x:Uid="PowerLauncher_ActionKeyword"
+                                                IsEnabled="{x:Bind Enabled, Mode=OneWay}">
+                                                <TextBox
+                                                    MinWidth="{StaticResource SettingActionControlMinWidth}"
+                                                    Text="{x:Bind Path=ActionKeyword, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
+                                            </labs:SettingsCard>
+                                            <labs:SettingsCard
+                                                Margin="0,0,0,0"
+                                                Padding="0,-4,0,0"
+                                                HorizontalContentAlignment="Left"
+                                                Background="{ThemeResource SystemFillColorCriticalBackgroundBrush}"
+                                                ContentAlignment="Vertical">
+                                                <InfoBar
+                                                    x:Uid="Run_NotAccessibleWarning"
+                                                    Margin="3,0,0,0"
+                                                    Background="Transparent"
+                                                    BorderBrush="Transparent"
+                                                    IsClosable="False"
+                                                    IsOpen="True"
```

Comment:
> The error of defining activation command is always on for PowerToys Run plugins

Because it's hardcoded?

Reply:
Fixed!

Reply:
Niels, this is circling back to the problem of "it is always on". The condition is required for the visibility.